es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 
 <br>
 
 **21. Types ― There are different sorts of hierarchical clustering algorithms that aims at optimizing different objective functions, which is summed up in the table below:**
 
-&#10230;
+&#10230; Tipos - Hay diferentes tipos de algoritmos de agrupamiento jerárquico que apuntan a optimizar diferentes funciones objetivo, que se resumen en la tabla a continuación:
 
 <br>
 
 **22. [Ward linkage, Average linkage, Complete linkage]**
 
-&#10230;
+&#10230; [Enlace Ward, Enlace promedio, Enlace completo]
 
 <br>
 
 **23. [Minimize within cluster distance, Minimize average distance between cluster pairs, Minimize maximum distance of between cluster pairs]**
 
-&#10230;
+&#10230; [Minimizar dentro de la distancia del clúster, Minimizar la distancia promedio entre pares de clúster, Minimizar la distancia máxima entre pares de clúster]
 
 <br>
 
 **24. Clustering assessment metrics**
 
-&#10230;
+&#10230; Métricas de evaluación de agrupamiento
 
 <br>
 
 **25. In an unsupervised learning setting, it is often hard to assess the performance of a model since we don't have the ground truth labels as was the case in the supervised learning setting.**
 
-&#10230;
+&#10230; En un entorno de aprendizaje no supervisado, a menudo es difícil evaluar el rendimiento de un modelo ya que no contamos con las etiquetas verdaderas, como es el caso del aprendizaje supervisado.
 
 <br>
 
 **26. Silhouette coefficient ― By noting a and b the mean distance between a sample and all other points in the same class, and between a sample and all other points in the next nearest cluster, the silhouette coefficient s for a single sample is defined as follows:**
 
-&#10230;
+&#10230; Coeficiente de silueta - Sea a y b la distancia media entre una muestra y todos los demás puntos en la misma clase, y entre una muestra y todos los demás puntos en el siguiente grupo más cercano, el coeficiente de silueta s para una muestra individual se define de la siguiente manera:
 
 <br>
 
 **27. Calinski-Harabaz index ― By noting k the number of clusters, Bk and Wk the between and within-clustering dispersion matrices respectively defined as**
 
-&#10230;
+&#10230; Índice de Calinski-Harabaz - Sea k el número de conglomerados, Bk y Wk las matrices de dispersión entre y dentro de la agrupación, respectivamente, se definen como:
 
 <br>
 
 **28. the Calinski-Harabaz index s(k) indicates how well a clustering model defines its clusters, such that the higher the score, the more dense and well separated the clusters are. It is defined as follows:**
 
-&#10230;
+&#10230; El índice de Calinski-Harabaz s(k) indica qué tan bien un modelo de agrupamiento define sus grupos, de tal manera que cuanto mayor sea el puntaje, más denso y bien separados estarán los conglomerados. Se define de la siguiente manera:
 
 <br>
 
 **29. Dimension reduction**
 
-&#10230;
+&#10230; Reducción de dimensión
 
 <br>
 
 **30. Principal component analysis**
 
-&#10230;
+&#10230; Análisis de los componentes principales
```

Comment:
Drop "los". Is usually called "_[Análisis de componentes principales](https://en.wikipedia.org/wiki/Ward%27s_method#cite_note-1)_".